hm_1/tasks/task_07_03.py

```diff
@@ -0,0 +1,42 @@
+"""
+Given a Tic-Tac-Toe 3x3 board (can be unfinished).
+Write a function that checks if the are some winners.
+If there is "x" winner, function should return "x wins!"
+If there is "o" winner, function should return "o wins!"
+If there is a draw, function should return "draw!"
+If board is unfinished, function should return "unfinished!"
```

Comment:
Please pay attention to the small details:  you return `unfinished` and task require `unfinished!`.

Sometimes even small details can provide a serious impact: `100$` and `900$` have a difference in a single char.